src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -232,6 +232,9 @@ void grpc_chttp2_ref_transport(grpc_chttp2_transport* t) { gpr_ref(&t->refs); }
 
 static const grpc_transport_vtable* get_vtable(void);
 
+// -1 == unset, 0 == disabled, 1 == enabled
+static gpr_atm flow_control_enabled = -1;
```

Comment:
should be `(gpr_atm)-1` for consistency with other uses, or better `flow_control_enabled(-1)`

Reply:
Your suggestion is much cleaner... Done and done